go.sum

```diff
@@ -0,0 +1,633 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
I'm not super familiar with go, but is it possible for this to be contained in the tests directory to keep that all together?

Reply:
yes! It was created in root more as a force of habit, but in this case where the only Go code is inside the tests folder, no problem in having it there.